sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/HelloWorldSamples.cs

```diff
@@ -43,22 +38,17 @@ public void HelloWorld()
             #region Snippet:GetStatus
 
             Response statusResponse = ledgerClient.GetTransactionStatus(transactionId);
-
-            string status = JsonDocument.Parse(statusResponse.Content)
-                .RootElement
-                .GetProperty("state")
-                .GetString();
+            dynamic transactionStatus = statusResponse.Content.ToDynamic();
+            string status = transactionStatus.state;
 
             Console.WriteLine($"Transaction status: {status}");
 
             // Wait for the entry to be committed
             while (status == "Pending")
             {
                 statusResponse = ledgerClient.GetTransactionStatus(transactionId);
-                status = JsonDocument.Parse(statusResponse.Content)
-                    .RootElement
-                    .GetProperty("state")
-                    .GetString();
+                transactionStatus = statusResponse.Content.ToDynamic();
+                status = transactionStatus.state;
```

Comment:
shouldn't we sleep at the end of loops like this?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/34085